packages/truffle-compile/profiler.js

```diff
@@ -372,29 +369,33 @@ module.exports = {
     });
   },
 
-  getImports: function(file, resolved, solc) {
-    var self = this;
+  getImports(file, { body, source }, solc) {
+    const self = this;
 
-    var imports = Parser.parseImports(resolved.body, solc);
+    // No imports in vyper!
+    if (path.extname(file) === ".vy") return [];
```

Comment:
These explicit checks are definitely technical debt, but I support acquiring this debt right now. Very nicely judicious!